fortnite_bot.py

```diff
@@ -175,10 +181,8 @@ async def stats_diff_today(ctx, usernames):
 
 async def opponent_stats_today(ctx):
     """ Outputs the stats of the players faced today """
-    # TODO: Wrap this up
-    res = await stats.get_opponent_stats_today()
-    print(res)
-
+    # TODO: 
```

Comment:
What's the TODO task remaining here?

Reply:
not sure if u wanted to refactor later, if not can delete